test/drivers/test_suites/syscall_enter_suite/ugetrlimit_e.cpp

```diff
@@ -0,0 +1,48 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_ugetrlimit
+
+#include <sys/resource.h>
+
+TEST(SyscallEnter, ugetrlimitE)
+{
+	/* Please note:
+	 * the syscall `ugetrlimit` is mapped to `PPME_SYSCALL_GETRLIMIT_E` event
+	 * like `getrlimit`. The same BPF program will be used for both the syscalls.
+	 */
```

Comment:
right now we don't have a dedicated event pair for `ugetrlimit` so we use the same fillers of `getrlimit`. This issue is tracked here https://github.com/falcosecurity/libs/issues/911